tool-openssl/x509.cc

```diff
@@ -279,15 +272,8 @@ bool X509Tool(const args_list_t &args) {
       }
     }
 
-    if (!out_path.empty()) {
-      if (!WriteSignedCertificate(x509.get(), out_path)) {
-        return false;
-      }
-    }
-
-    if (!noout && !in_path.empty() && !checkend && parsed_args.count("-out")==0) {
-      bssl::UniquePtr<BIO> bio_out(BIO_new_fp(stdout, BIO_NOCLOSE));
-      if (!PEM_write_bio_X509(bio_out.get(), x509.get())) {
+    if (!noout && !checkend && !modulus) {
```

Comment:
we didn't check `modulus` previously here. why do it now?

Reply:
I was trying to fix a failing test, turns out OpenSSL 1.0.2 and 1.1.1 have wildly different behavior of what gets written to the -out parameter vs what is always written to stdout. I think testing against 1.0.2 has very limited benefit and is hurting our testing against newer versions.